### ISSUE TITLE
Feature Request: Preview mode and confirm while scanning

### ISSUE THREAD

**Issue by eugenioghio** [phonegap/phonegap-plugin-barcodescanner#827]
### Current Behaviour
_After a code is recognized, the camera is shut after the resultDisplayDuration time is expented and the scanned text is returned._

All good and dandy. 
As long it's a Qr code, but barcodes often came in copious hosts. 

As a warehouse worker you find yourself in need to confirm the barcode before shutting the camera, as you may just have scanned the wrong one in a page of one hundred barcodes.

All the consequent: 
- wait-for-the-camera-to-shut, 
- realized that the code was the wrong one because of that wrong figure in 30 chars long code, 
- press the scan button again, 
- wait-for-the-camera-to-spool-up again
- and scan **again**

**is time expensive**.

### Desired Behaviour
If the brand new scanInBulk option is set to true then:
_After a code is recognized and displayed, the camera wont shut because of the countdown, instead it waits to return the text until a button of confirm is pressed , in the mean time if another code is centered on the scan rectangle **the code is updated with the new one** and displayed_

(said confirm button must be on the bottom of the screen at the center, on camera, like the torch button)

Basically it is a preview - display and confirm routine that solve a find needle in haystack situation.

It may not sound like a necessary features, but I know about scenarios in which it would save time and made work smoother.
